Build error on building julia with USE_INTEL_JITEVENTS = 1

```
$ make
    CC src/threading.o
/home/kristoffer/juliaintel/src/threading.c(544): error: identifier "sig_stack_size" is undefined
              sig_stack_size = 1<<16;
              ^

compilation aborted for /home/kristoffer/juliaintel/src/threading.c (code 2)
make[1]: *** [threading.o] Error 2
make: *** [julia-src-release] Error 2
```

CC @ArchRobison

--- a/src/julia_internal.h
+++ b/src/julia_internal.h
@@ -14,6 +14,9 @@ extern size_t jl_page_size;
 extern char *jl_stack_lo;
 extern char *jl_stack_hi;
 extern jl_function_t *jl_typeinf_func;
+#if defined(JL_USE_INTEL_JITEVENTS)
+extern unsigned sig_stack_size;
+#endif
 
 DLLEXPORT extern int jl_lineno;
 DLLEXPORT extern const char *jl_filename;

